nc/source/graphics/Graphics2.cpp

```diff
@@ -142,34 +133,34 @@ namespace nc::graphics
 
     bool Graphics2::GetNextImageIndex(uint32_t& imageIndex)
```

Comment:
Will you make imageIndex a uint32_t* so it is clear it is an out param at call site?